DEVELOPMENT.md

```diff
@@ -4,15 +4,51 @@
 > to find out about how to contribute to Flux and how to interact with the
 > Flux Development team.
 
+## Installing required dependencies
+There are a number of dependencies required to be able to run image-reflector-controller and its test-suite locally. 
+* [Install Go](https://golang.org/doc/install)
+* [Install Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+* [Install Docker](https://docs.docker.com/engine/install/)
+* (Optional) [Install Kubebuilder](https://book.kubebuilder.io/quick-start.html)
+
 ## How to run the test suite
 
 Prerequisites:
 * go >= 1.16
-* kubebuilder >= 2.3
 * kustomize >= 3.1
 
-You can run the unit tests by simply doing
+Prior committing any changes, especially when API changes are made, please run unit tests. You can run them by simply doing
```

Comment:
Maybe keep the original and just add the note after the `make test` code block.

Something like this, also explains the reason behind doing that:
> Since this will also trigger generating some files such as manifests, it is advised to run this prior to committing your changes, especially when making API changes.